omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -41,6 +41,9 @@
 
 default['private_chef']['fips_enabled'] = ChefConfig.fips?
 
+default['private_chef']['required_recipe']['enable'] = false
```

Comment:
But if the file is stored in the database, then we'll need no config because if it's there it should be returned and if it's not it wont. But that's jumping to implementation details.